packages/components/data-table-manager/README.md

```diff
@@ -53,18 +56,19 @@ const columns = [
 ];
 
 const Example = () => (
-  <DataTableManager columns={columns}>
+  <DataTableManagerProvider>
```

Comment:
I think we need to provide two different examples. The current one and another one with the detached behaviour where the `DataTable` and `DataTableManager` should be in different component's tree leaves.

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/bd8447871f0d99ac366b5adc9d865055c9a4506e

Thank you